eahub/profiles/management/commands/profilesfromlegacyhub.py

```diff
@@ -12,6 +12,61 @@
 from ....base import models as base_models
 
 
+def collect_cause_areas(
+    global_poverty,
+    animal_welfare_and_rights,
+    long_term_future,
+    cause_prioritisation,
+    meta,
+):
+    cause_areas = []
+    if global_poverty:
+        cause_areas.append(models.CauseArea.GLOBAL_POVERTY)
+    if animal_welfare_and_rights:
+        cause_areas.append(models.CauseArea.ANIMAL_WELFARE_AND_RIGHTS)
+    if long_term_future:
+        cause_areas.append(models.CauseArea.LONG_TERM_FUTURE)
+    if cause_prioritisation:
+        cause_areas.append(models.CauseArea.CAUSE_PRIORITISATION)
+    if meta:
+        cause_areas.append(models.CauseArea.META)
+    return cause_areas
+
+
+CODED_AREAS_BY_LEGACY_NAME = {
+    "Coding": models.ExpertiseArea.SOFTWARE_ENGINEERING,
+    "Entrepreneurship": models.ExpertiseArea.ENTREPRENEURSHIP,
+    "Event planning": models.ExpertiseArea.EVENT_PLANNING,
+    "Finance/Investment strategy": models.ExpertiseArea.FINANCE,
+    "Graphic design": models.ExpertiseArea.GRAPHIC_DESIGN,
+    "Journalism": models.ExpertiseArea.JOURNALISM,
+    "Law": models.ExpertiseArea.LAW,
+    "Management": models.ExpertiseArea.MANAGEMENT,
+    "Marketing/Sales": models.ExpertiseArea.COMMUNICATIONS,
```

Comment:
Is that the same thing? Marketing/Sales = Communications?

Reply:
Ignore this for now, it's part of #479.